pkg/client/cache/listwatch.go

```diff
@@ -31,16 +32,16 @@ import (
 type ListerWatcher interface {
 	// List should return a list type object; the Items field will be extracted, and the
 	// ResourceVersion field will be used to start the watch in the right place.
-	List(options api.ListOptions) (runtime.Object, error)
+	List(options v1.ListOptions) (runtime.Object, error)
```

Comment:
If it stays internal version, then we'll need to convert the ListOptions to v1 before passing it to the client's List/Watch methods. Is using v1 wrong?